### PR TITLE
main.cpp

### PR DESCRIPTION
**Pull request by chenlian2015**
eosc  pamameter limit  not work

### CODE REVIEW

**Comment by chenlian2015** [comment]
eosc  pamameter limit  not work

File: programs/eosioc/main.cpp [hunk 1]
```diff
@@ -560,6 +560,7 @@ int main( int argc, char** argv ) {
    string table;
    string lower;
    string upper;
+   string key;
```
Comment (wanderingbort) [inline programs/eosioc/main.cpp]: This appears to go completely unused.

Comment (chenlian2015) [reply programs/eosioc/main.cpp]: already used!! 
eg.
eosc      get table oct oct account
eosc      get table songbo oct account
eosc      get table zhaobo oct account
when account table rows large enough, limit become import !!!

**Comment by heifner** [comment]
This was fixed via DAWN-245.

**Pull request closed without merging**